### ISSUE TITLE
Image block - Intentionally missing Top Wave and Bottom Wave styles?

### ISSUE THREAD

**Issue by formosattic** [Automattic/wp-calypso#45267]
The **Top Wave** or **Bottom Wave** options come with **CoBlocks**, they are made available by default in the **Cover block**, but are missing in the **Image block** (unless site is atomic), is that expected?

### Cover block - Free plan
![Markup on 2020-08-28 at 16:09:38](https://user-images.githubusercontent.com/15944096/91537447-e65d1b00-e948-11ea-953d-461db935aa65.png)

### Image block - Free plan

![Markup on 2020-08-28 at 16:10:16](https://user-images.githubusercontent.com/15944096/91537540-07be0700-e949-11ea-9272-c5c1897f33b7.png)

### Image block - WordPress.org site

![Markup on 2020-08-28 at 16:13:54](https://user-images.githubusercontent.com/15944096/91537834-7c914100-e949-11ea-95e2-0c5eda3da602.png)

**Comment by synora** [comment]
Yes, this does appear to be expected behavior. I've added a feature request tag to this issue.
